CHANGELOG.md

```diff
@@ -19,6 +19,10 @@ The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),
     - or else EhrComposition effectiveTime - high
     - or else EhrComposition effectiveTime - low
 
+### Fixed
```

Comment:
This feels like a breaking change, because NMEs using the adaptor will need to update their importing routines to look for this new "system" value.

I'm aware that we have this URN/Not a URN issue elsewhere.

If we were to make this breaking change, I'd suggest we make all the breaking changes at once to minimise pain for our users.

Thoughts?

Reply:
This has now been addressed, once this pull request has been approved / merged, I will need to let Emile know to ready the changes on the Medicus side for release